packages/components/src/components/Image/Image.css

```diff
@@ -2,6 +2,8 @@
 
 .cf-no-image {
   position: relative;
+  height: 100%;
```

Comment:
wouldn't we use the styling that comes from the component bindings for height and width, even if there was no image?

Reply:
We do apply the Image size when there is no image bound.  The new `<PreviewUnboundImage>` component handles that, and applies the image size styles to a wrapper element.

Your comment made me realize that we won't really need this `.cf-no-image` element any more.  Previously this wrapper was used for centering the svg icon within the placeholder element.  With the new `<PreviewUnboundImage>` component providing a wrapper, this `.cf-no-image` wrapper can go away.  I refactored this a little bit and added some classes so it's easier to find where each element comes from in the codebase.

![Screenshot 2024-10-02 at 3 20 00 PM](https://github.com/user-attachments/assets/d88e62ee-03f8-4c83-bd90-02659af9398e)

* `.cf-preview-unbound-image` is the wrapper in `<PreviewUnboundImage>` which receives the image size styles
* `.cf-placeholder-image` and `.cf-placeholder-icon` are the placeholder elements in the `<Image>` component when no image is bound